src/modules/billing/services/supplementary-billing-service/supplementary-processor.js

```diff
@@ -73,7 +73,7 @@ const getGroupingKey = transaction => {
   // Get the data and serialize to a JSON string
   const data = {
     ...pick(transaction, keys),
-    description: transaction.description.trim().toLowerCase(),
+    description: transaction.chargeElementPurposeUseCode,
```

Comment:
Would you mind changing the `description` field here to another name, such as `chargeElementPurposeUseCode` - just since this isn't actually pulling the description any more